loom/graphics/gfxVectorRenderer.cpp

```diff
@@ -32,6 +32,10 @@
 
 #include "loom/script/runtime/lsProfiler.h"
 
+#define NVG_malloc(sz) lmAlloc(NULL, sz)
+#define NVG_realloc(sz) lmRealloc(NULL, sz)
+#define NVG_free(sz) lmFree(NULL, sz)
+
 #include "nanovg.h"

```

Comment:
If it was compiled on its own compilation unit then it would call malloc instead of lmAlloc. But you checked this so it's good I think?